src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -662,4 +665,16 @@ public static double stddev(final double[] in, final int start, final int stop,
         for (int i = 0; i < is.length; ++i) ds[i] = is[i];
         return ds;
     }
+
+    /**
+     * Compute the median of a list of numbers
+     * @param values a list of numbers
+     * @return the median element
+     */
+    public static <T extends Number & Comparable<T>> double median(final Collection<T> values) {
```

Comment:
as a heads up, the old version of HomRefBlock used this [implementation of median](https://github.com/broadinstitute/gsa-unstable/blob/master/public/gatk-utils/src/main/java/org/broadinstitute/gatk/utils/MathUtils.java#L975-L1002) which is not actually the median.  I've changed to the proper calculation of the median, which WILL produce different results.  It should hopefully be producing a better result, but it's also possible the other was intentional.